### ISSUE TITLE
Support LVM snapshots

### ISSUE THREAD

**Issue by baztian** [teejee2008/timeshift#82]
Timeshift looks awesome. Can you please add support for lvm2 snapshots?

Thanks

Bastian

**Comment by pepa65** [comment]
That would be awesome! I use lvm2 on all my systems, and this would make it much easier to leverage the inbuild snapshot functionality.

**Comment by moocan** [comment]
Yes that would be really awesome if this great software could use lvm snapshot functionality . I use lvm2 everywhere too (and never again btrfs due to many corruption).
